rocky/rocky/templates/tasks/partials/task_actions.html

```diff
@@ -13,18 +13,25 @@ <h2>{% translate "Yielded objects" %}</h2>
         {% if task.status.value in "completed,failed" %}
             {% if task.type == "normalizer" %}
                 <a class="button"
-                   href="{% url 'bytes_raw' organization_code=organization.code boefje_meta_id=task.data.raw_data.boefje_meta.id %}"><span class="icon ti-download"></span>{% translate "Download meta and raw data" %}</a>
-                {% include "partials/single_action_form.html" with btn_text=_("Reschedule") btn_class="ghost" btn_icon="icon ti-refresh" action="reschedule_task" key="task_id" value=task.id %}
+                   href="{% url 'bytes_raw' organization_code=task.data.raw_data.boefje_meta.organization boefje_meta_id=task.data.raw_data.boefje_meta.id %}"><span class="icon ti-download"></span>{% translate "Download meta and raw data" %}</a>
```

Comment:
Not new in this PR, but do these icons use an icon font and pseudo-content? If so, they should either get an `aria-label` or `aria-hidden="true"` - most likely the latter since the links already have accessible text.

Reply:
As this is not new in the PR, I would like to suggest raising a separate issue for this. I also think this may be out of scope for a minimum viable product

Reply:
Ticket: https://github.com/minvws/nl-kat-coordination/issues/3404